composer.json

```diff
@@ -1,29 +1,31 @@
 {
     "name": "zendframework/zend-stdlib",
-    "description": " ",
+    "description": "SPL extensions, array utilities, error handlers, and more",
+    "type": "library",
```

Comment:
Can be removed, because `library`is the [default value for type](https://getcomposer.org/doc/04-schema.md#type).